src/sql_engine/src/ddl/create_table.rs

```diff
@@ -46,12 +46,15 @@ impl<P: BackendStorage> CreateTableCommand<P> {
                 .map(|c| {
                     let name = c.name.to_string();
                     let sql_type = match c.data_type {
-                        sqlparser::ast::DataType::SmallInt => SqlType::SmallInt,
-                        sqlparser::ast::DataType::Int => SqlType::Integer,
-                        sqlparser::ast::DataType::BigInt => SqlType::BigInt,
+                        sqlparser::ast::DataType::SmallInt => SqlType::SmallInt(i16::min_value()),
+                        sqlparser::ast::DataType::Int => SqlType::Integer(i32::min_value()),
+                        sqlparser::ast::DataType::BigInt => SqlType::BigInt(i64::min_value()),
                         sqlparser::ast::DataType::Char(len) => SqlType::Char(len.unwrap_or(255)),
                         sqlparser::ast::DataType::Varchar(len) => SqlType::VarChar(len.unwrap_or(255)),
                         sqlparser::ast::DataType::Boolean => SqlType::Bool,
+                        sqlparser::ast::DataType::Custom(ObjectName(_serial)) => SqlType::Integer(1),
```

Comment:
```suggestion
                        sqlparser::ast::DataType::Custom(name) => {
                            let name = name.to_string();
                            if name == "smallserial" {
                                SqlType::SmallInt(1);
                            } else if name == "serial" {
                                SqlType::Integer(1);
                            } else if name == "bigserial" {
                                SqlType::BigInt(1);
                            } else {
                                unimplemented!("{:?} type is not supported", name)
                            }
                        },
```

Reply:
this should help

Reply:
Thanks  @alex-dukhno fixed, please review it.